packages/buckets/src/buckets.ts

```diff
@@ -52,6 +52,7 @@ export interface PushPathResult {
  * ```
  */
 export class Buckets {
+  private _client: Client
```

Comment:
this is starting to mirror more closely buckd, where buck has most of threads capabilities. wanted to simplify the context moving for buckets, so just hanging onto the private instance here.